### ISSUE TITLE
[8.x] ORA-00928 when using Model::create() with guarded property.

### ISSUE THREAD

**Issue by planetchili** [yajra/laravel-oci8#596]
### Summary of problem or feature request

Cannot use ::create to insert a new model row with OracleEloquent.

### Code snippet of problem

This code yields the error: ORA-00928: SELECT keyword missing
```php
MiccStackItem::create($params);
```
The SQL query that is executed: insert into "MICC_STACK_ITEMS" () values () returning "ID" into :p0 Bindings : [0] "

This code works fine:
```php
$item = new MiccStackItem;
$item->data = $params['data'];
$item->save();
```

The model:
```php
use Yajra\Oci8\Eloquent\OracleEloquent as Model;

class MiccStackItem extends Model
{
    public $timestamps = false;
    protected $guarded = ['id'];
    protected $connection = 'oracle';
}
```

The migration:
```php
Schema::connection('oracle')
      ->create('micc_stack_items', function (Blueprint $table) {
      $table->increments('id');
      $table->string('data');
});
```

### System details

- Operating System: Ubutu Focus
- PHP Version: 7.4
- Laravel Version: 8
- Laravel-OCI8 Version: 8.0.1

**Comment by yajra** [comment]
Thanks for reporting, I was able to replicate the issue.

On the other hand, I think think has something to do with the guarded. Try setting `protected $guarded = [];` and create would work just fine.

**Comment by yajra** [comment]
Does the same code works on mysql or other dbms? Extending either the Model or OracleEloquent have the same error.

**Comment by yajra** [comment]
After playing around on PR #609, It appears like a bug in the framework and not on the package. Will try to send a PR and hope it gets merged 🤞.

**Comment by yajra** [comment]
Just realized after sending the PR that it seems like an intended framework behavior. Forcing the column listing to lower case would fix this issue. 

Patch released on v8.1.3. 🚀
